packages/gatsby/src/schema/graphql-engine/standalone-regenerate.ts

```diff
@@ -26,6 +26,7 @@ import { store } from "../../redux"
 import { validateEngines } from "../../utils/validate-engines"
 
 async function run(): Promise<void> {
+  process.env.GATSBY_SLICES = `1`
```

Comment:
Double checking, is this change also required for the fix or left in accidentally?

Reply:
This is not needed for a fix as we don't actually use this file in regular usage. 

I just noticed problems with trying to use engines regenerated with this script manually as I was iterating on this PR and testing produced engines